docs/ja/saas/tenant-admin-user.md

```
@@ -0,0 +1,139 @@
+```
+
+成功した場合の出力例：
+
+```json
+[
+  { "id": "my-company", "displayName": "My Company", "createdAt": "2026-01-01T00:00:00.000Z" }
+]
+```
+
+::: details テナント作成でエラーが出た場合
+| エラー | 原因 | 対処 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)